src/exc.ts

```diff
@@ -50,6 +50,12 @@ export class CannotPerformOperationError extends AppError {
   }
 }
 
+export class ObjectValidationError extends AppError {
+  constructor(m: string, status = 400) {
+    super(m, status);
+    Object.setPrototypeOf(this, CannotPerformOperationError.prototype);
```

Comment:
small copy paste fix - change `CannotPerformOperationError` to `ObjectValidationError`